lib/types.js

```diff
@@ -61,6 +60,24 @@ TInt.prototype = {
 
 exports.TInt = TInt;
 
+function TLong (value) { this.value = value; }
+TLong.prototype = {
+    typetag: 'h',
+    decode: function (data) {
+        if (data.length < 8) {
+            throw new ShortBuffer('int', data, 8);
+        }
+
+        this.value = jspack.Unpack('>i', data.slice(0, 8))[0];
+        return data.slice(8);
+    },
+    encode: function (buf, pos) {
+        return pack('>i', buf, pos, [ this.value ]);
```

Comment:
the jspack docs specify that a `long` should include the format character `l` for signed and `L` for unsigned. Putting `i` here will not work as expected